dev/js/wow_wizard.js

```diff
@@ -378,6 +318,30 @@
 			}
 		}
 
+		// If the step has steps depending on it, function returns array id's of them.
+		function _clearFutureSteps($step) {
+			var depending_steps = [];
+			for(var i = currentStep+1; i <= passedStepTracker; i++) {
+				var $step_filled = wizard.settings.steps[i];
+				if($step_filled.isDependent) {
+					var found = false;
+					for(var j = 0; j < $step_filled.steps.length && !found; j++) {
```

Comment:
If for loop terminates when it's found, what is the point of returning an array?